thingif/src/main/java/com/kii/thingif/command/Action.java

```diff
@@ -30,5 +24,12 @@
  *
  */
 public interface Action {
+    /**
```

Comment:
:)
> If we make Action as abstract class, we can check it inside of this class.

I can't understand whey checking getActionName() needs abstract class.  I used this way to check the return value of getActionName method from [this line](https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/293/files#diff-12e2b8b1411bb1a56fce72fd0a2187a8R270).